espnet2/spk/pooling/chn_attn_stat_pooling.py

```diff
@@ -0,0 +1,54 @@
+import torch
+import torch.nn as nn
+
+from espnet2.spk.pooling.abs_pooling import AbsPooling
+
+
+class ChnAttnStatPooling(AbsPooling):
+    """
+    Aggregates frame-level features to single utterance-level feature.
+    Proposed in B.Desplanques et al., "ECAPA-TDNN: Emphasized Channel
+    Attention, Propagation and Aggregation in TDNN Based Speaker Verification"
+
+    """
+
+    def __init__(
+        self,
+        input_size: int = 1536,
+        output_size: int = 3072,
+    ):
+        super().__init__()
+        self.attention = nn.Sequential(
+            nn.Conv1d(input_size * 3, 128, kernel_size=1),
+            nn.ReLU(),
+            nn.BatchNorm1d(128),
+            nn.Conv1d(128, input_size, kernel_size=1),
+            nn.Softmax(dim=2),
+        )
+
+    def output_size(self) -> int:
+        return self._output_size
+
+    def forward(self, x):
```

Comment:
Could you add a docstring for this method? For example, describe the input and output.